src/pages/Tools/constants/testnet.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { predefined, type ScriptConfigs, type Config } from '@ckb-lumos/config-manager'
+
+const TESTNET_SCRIPTS: ScriptConfigs = {
```

Comment:
These lock script info can be found at https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/constants/scripts.ts